src/Settings/constants.js

```diff
@@ -0,0 +1,7 @@
+export const SETTINGS_API = 'organizations-storage/settings';
+export const BANKING_ACCOUNT_TYPES_API = 'organizations-storage/banking-account-types';
+
+export const BANKING_INFORMATION_SEARCH_QUERY = {
+  query: 'key=BANKING_INFORMATION_ENABLED',
```

Comment:
It still seems to me that the `BANKING_INFORMATION_ENABLED` value should lay separately